src/scribe.js

```diff
@@ -192,14 +203,28 @@ define([
        */
 
       // We only want to push the history if the content actually changed.
-      if (! previousUndoItem || (previousUndoItem && this.getHTML() !== previousContent)) {
+      if (! previousUndoItem || (previousUndoItem && this.getHTML() !== previousContentNoMarker)) {
         var selection = new this.api.Selection();
 
         selection.placeMarkers();
         var html = this.getHTML();
         selection.removeMarkers();
 
-        this.undoManager.push(html);
+        // Register a new change transaction.
+        var scribe = this;
```

Comment:
Not a blocker but having bound this to a name I think it would be better to use it on the next line as well